book/src/overview.md

```diff
@@ -74,13 +74,13 @@ This block will error
 This block will error
 ```
 
-You can also configure the build to fail loudly, by setting `on_failure = "bail"` in `book.toml`. See the [configuration reference](./reference.md) for more details.
+You can also configure the build to fail loudly, by setting `on_failure = "bail"` in `book.toml`. See the [configuration reference](./reference.md#booktoml-configuration) for more details.
 
 ### Additional Options
 
 You can pass additional options to each block. The options are structured as TOML key-value pairs.
 
-Note that some options can be passed globally, through the `default` section in `book.toml`. See the [configuration reference](./reference.md) for more details.
+Note that some options can be passed globally, through the `default` section in `book.toml`. See the [configuration reference](./reference.mdbooktoml-configuration) for more details.
```

Comment:
I think you're missing a `#` here
```suggestion
Note that some options can be passed globally, through the `default` section in `book.toml`. See the [configuration reference](./reference.mdbook#toml-configuration) for more details.
```

Reply:
Oh yes somehow missed that. But I think the `#` belongs after the `.md` file extension.

```diff
- ./reference.mdbook#toml-configuration
+ ./reference.md#booktoml-configuration
```

Links to headers do not include backticks (`` ` ``) or periods (`.`) which is the reason why it looks that weird.

Reply:
Ah of course, thanks